haproxy/metadata.csv

```diff
@@ -50,8 +50,8 @@ haproxy.sticktable.used,gauge,,item,,Number of elements in the table.,0,haproxy,
 haproxy.sticktable.size,gauge,,item,,Maximum number of elements the table can hold.,0,haproxy,stick-table max size
 haproxy.backend.active.servers,gauge,,,,Current number of active servers.,0,haproxy,
 haproxy.backend.backup.servers,gauge,,,,Current number of backup servers.,0,haproxy,
-haproxy.backend.bytes.in.total,gauge,,byte,,Current total of incoming bytes.,0,haproxy,
-haproxy.backend.bytes.out.total,gauge,,byte,,Current total of outgoing bytes.,0,haproxy,
+haproxy.backend.bytes.in.total,gauge,,byte,,Current total of incoming bytes. Submitted as count if using prometheus,0,haproxy,
```

Comment:
```suggestion
haproxy.backend.bytes.in.total,gauge,,byte,,Current total of incoming bytes. By default submitted as count if using prometheus,0,haproxy,
```
The submitted type depends on the prometheus integration configuration, should we be a bit more precise here?